src/renderer/Presentational/NodeScreen/NodeScreen.tsx

```diff
@@ -377,19 +378,19 @@ const NodeScreen = () => {
     onAction: onNodeAction,
   };
   console.log('passing content to NodeScreen: ', nodeContent);
-  return <ContentSingleClient {...nodeContent} />;
+  return (
+    <div>
```

Comment:
We might need to use <ContentHeader> component here instead of the backbutton.

Sample code:

```
<ContentHeader
            leftButtonIconId="left"
            leftButtonOnClick={() => navigate('/main/nodePackage')}
          />
```

The component might need to be modified to support no text at all 🤔

Reply:
Just checked and functionality and styles for this seem different than ContentHeader